core/src/main/java/oracle/weblogic/deploy/util/XPathUtil.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright (c) 2021, Oracle Corporation and/or its affiliates.
```

Comment:
Since this change did not make it into the main line by the end of the year, each of the copyrights needs to be changed to 2022.